### ISSUE TITLE
[FEAT] `wadm.api.{lattice}.model.list` should return the response wrapper type

### ISSUE THREAD

**Issue by lachieh** [wasmCloud/wadm#278]
At the moment, requests on the `wadm.api.{lattice}.model.list` topic return a list of model summaries. All the other api topics return a response wrapper with the shape below. The `list` operation should return the same wrapper.

```
{
   status: 'success' | 'error',
   message: string,
   ...
}
```

**Comment by lachieh** [comment]
Rather than changing the existing api, @brooks suggested that we do the k8s thing and add a `model.get` in addition to `model.get.<app_name>` that responds with the expected wrapper type and deprecate the `model.list` topic.

**Comment by lachieh** [comment]
not stale

**Comment by lachieh** [comment]
Closed by #400
